### DISCUSSION TITLE
新版本g2 5.2.1 没有保存图片的方法了， chart.downloadImage()不能使用显示不存在这个方法

### DISCUSSION THREAD

**Discussion by XcLyyds00176859245146532** [antvis/G2#6376] [Q&A]
### 问题描述

新版本g2 5.2.1 没有保存图片的方法了， chart.downloadImage()不能使用显示不存在这个方法，希望能像属性一样配置即可有下载功能
![image](https://github.com/user-attachments/assets/f875d366-ee3e-4406-a091-1815e10719aa)

### 预期行为

希望可以通过属性配置就能拥有下载图表为图片的功能

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by pearmini** [comment]
暂时没有提供这个方法的计划，可以通过拿到渲染的 canvas 然后去下载：

```js
const canvas = container.querySelector('canvas');
// 下载代码
```
